### PR TITLE
JIRA: EDGOAIPMH-11 - Edge OAI-PMH boilerplate files 

### PR DESCRIPTION
**Pull request by khandramai**
News, Readme, Contributors, License, Jenkinsfile, Dockerfile, etc.

Needed files added.

Viachaslau Khandramai <Viachaslau_Khandramai@epam.com>

### CODE REVIEW

File: NEWS.md [hunk 1]
```diff
@@ -0,0 +1 @@
+
```
Comment (epam-pavelk) [inline NEWS.md]: Let's add info about the version and initial commit here

File: pom.xml [hunk 2]
```diff
@@ -0,0 +1,189 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <groupId>org.folio</groupId>
+  <artifactId>edge-oai-pmh</artifactId>
+  <version>0.0.1-SNAPSHOT</version>
```
Comment (epam-pavelk) [inline pom.xml]: I guess the initial version should be 1.0.0

**Comment by khandramai** [comment]
@epam-pavelk,
done

File: Jenkinsfile [hunk 3]
```diff
@@ -0,0 +1,14 @@
+buildMvn {
+  publishModDescriptor = 'no'
+  publishAPI = 'yes'
```
Comment (piotr-kalashuk) [inline Jenkinsfile]: Probably for now it makes sense to set all actions to 'no' and enable once corresponding files are being added

**Comment by khandramai** [comment]
@piotr-kalashuk,
done.
Thanks!

**Comment by khandramai** [comment]
@dcrossleyau
thank you for notes. 
The needed changes have been applied.

**Pull request merged**